.github/workflows/build.yaml

```diff
@@ -18,3 +18,13 @@ jobs:
       - run: cp .env-dist .env
       # Verify that the build (incl. type-checking) succeeds
       - run: npm run build
+        env:
+          UPLOAD_SENTRY_SOURCEMAPS: false
+        if: ${{ !startsWith(github.ref, 'refs/tags/') }}
```

Comment:
Hypothetically speaking, if I didn't know what exactly this was doing/checking... what is this doing/checking?
Only uploading sourcemaps in stage/prod but not local/Heroku?

Reply:
Yes that's right... I'll add a comment.